src/components/BannerSlider.tsx

```diff
@@ -79,6 +82,15 @@ class BannerSlider extends React.PureComponent<BannerSliderPanelProps, BannerSli
 
   curryWithPage = (index: number) => () => (this.props.onPress ? this.props.onPress(index) : null);
 
+  onSetHeight(newHeight: number) {
+    if (
+      !this.state.heightImage ||
+      (this.state.heightImage && newHeight && newHeight > this.state.heightImage)
```

Comment:
I'd add a comment here to explain why the last comparison is `>` instead of `!==` (something like "all slides inside the slider will call this function as soon as they calculate their height, and we want the slider to be as tall as the tallest slide")